HW1_task1.cpp

```diff
@@ -0,0 +1,33 @@
+#include <iostream>
+#include <cmath>
+using namespace std;
+ 
+int main() {
+  /*
+  HOMEWORK
+ ax^2 + bx + c = 0
+ */
+
+int a, b, c, discriminant, x1, x2; 
+cout<<endl<<"Enter values: "<<endl<<"a = ";
+cin>>a;
+cout<<"b = ";
+cin>>b;
+cout<<"c = ";
+cin>>c;
+
+
+discriminant = (b*b)-(4*a*c);
```

Comment:
тут как раз пример того что тут стоит обьявлять переменную `discriminant` а не выше по коду.
```suggestion
const long long discriminant { (b*b)-(4*a*c) };
```